src/jsii-project.ts

```diff
@@ -329,7 +343,7 @@ export class JsiiProject extends TypeScriptProject {
     });
   }
 
-  private publishToMaven() {
+  private publishToMaven(java: any) {
```

Comment:
Keep a minimal surface area if possible:

```suggestion
  private publishToMaven(serverId: string | undefined, repositoryUrl: string | undefined) {
```